src/Cosi-BLS/range-proofs.lisp

```diff
@@ -693,421 +695,578 @@ nbits pts will become right vectors."
+                                             :Y 72078513108783763352610628977208112546724411941188746464434416021199666069356017217990638572189175995)
+                       #S(EDWARDS-ECC:ECC-PT :X 68878829257366500657600537957317810767067938419772208308261483456198368991805610563437180150881253762
+                                             :Y 84240518129441858178376516418653406809501179451493922655105715465844735826633970491935827027971908932)
+                       #S(EDWARDS-ECC:ECC-PT :X 98807538428893187936275935470652835152583250833931124820154806153940169341263689682888035610132443779
+                                             :Y 120858745001604708023993340274455796186014509886489170516561138228138634112912877276480568895622786514)
+                       #S(EDWARDS-ECC:ECC-PT :X 127684489935459095096201234314239555781679324726733350387458471739931169448070834643076317033151186663
+                                             :Y 120130730347953573919491840699516465749382375408980122451096740344380484928608159426884775371813727439)
+                       #S(EDWARDS-ECC:ECC-PT :X 65122639435984598026499444245384854033551094632330233053922807180410638012740903060424353147387477293
+                                             :Y 22803123397658753392702207956916501647041995484128114262506725576831949693736874957321026170126064104)
+                       #S(EDWARDS-ECC:ECC-PT :X 48210453660689983599420236249884458155323178737898937908665623286135810117898317672658054434721688878
+                                             :Y 3857846018960059521616150824043489679601914932598776495944310602925191660143638574328898156172897214)
+                       #S(EDWARDS-ECC:ECC-PT :X 105968152960684772834228102383082157621763978818583662212602933558827562069659217627078568099438070188
+                                             :Y 128382501823415793112357136167375348434491454667477837757722661790401332030810955470055559677194261363)
+                       #S(EDWARDS-ECC:ECC-PT :X 62859310690692104439190754696506425170686824923334672081364245153911201754012804608321252724805993020
+                                             :Y 3760421780060498487517447320615378370810044963886150660061928025278310740205627871036859005592267876)
+                       #S(EDWARDS-ECC:ECC-PT :X 15568750589062821236138807691374198983959891811555119805752749545706365617418647513300302985006512476
+                                             :Y 91505405360983251888250387839928569183865835366962010210822736086026219950642076015876943165831719768))))
+
+
+
+(defvar *chk-bp-basis*
+  ;; = (hex-str (HASH/256 *bp-basis*))
+  "A67E8B48750559A9CFA30B27E094E98DD8BC6BA22C1606AAECD7B9EA00E9B457")
+
+#||#
+;; ---------------------------------------------------------------------------
+;;
+;; for 128 bit security |#
 #|
 ;; Constants obtained from:
 
-       (with-ed-curve :curve-1174
+       (with-ed-curve :curve1174
```

Comment:
I kinda like the style of using `#\-` `for-separating-words` better, but not necessary to change this now.

Your usage here is certainly consistent with how the C code names things, which doesn't have the ability to use `#\-" as such a separator.

Reply:
My naming conventions follow the academic literature (or try to), so Curve1174 is its name in print, as is Ed3363, where I do separate as curve-Ed3363 in order to keep it clear in the code that it is an ECC curve.)

Reply:
Fine then.